### PR TITLE
Add compactor validator script [Needs more work - can't merge]

### PR DESCRIPTION
**Pull request by SravanthiAshokKumar**
## Overview

Description:

Why should this be merged: 

Related issue(s) (if applicable): #<number>

## Checklist (Definition of Done):

- [ ] There are no TODOs left in the code
- [ ] [Coding conventions](https://github.com/CorfuDB/CorfuDB/wiki/Corfu-Style-Guidelines) (e.g. for logging, unit tests) have been followed
- [ ] Change is covered by automated tests
- [ ] Public API has Javadoc

### CODE REVIEW

File: scripts/trigger_and_validate_compactor.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+
+"""
+Copyright 2023 VMware Inc. All rights reserved. VMware Confidential.
```
Comment (Lujie1996) [inline scripts/trigger_and_validate_compactor.py:4]: Remove this line

Comment (SravanthiAshokKumar) [reply scripts/trigger_and_validate_compactor.py]: Done

**Review by Lujie1996 — APPROVED** [review]

**Review by chetangudisagar — APPROVED** [review]

**Review by hisundar — COMMENTED** [review]
Would be nice to have a commit message explaining why this change is needed. thanks

**Review by hisundar — APPROVED** [review]
Looks good otherwise. thanks

**Pull request closed without merging**